package.json

```diff
@@ -140,7 +141,7 @@
     "pretty-format": "^29",
     "react-split-pane@^0.1.92": "patch:react-split-pane@npm%3A0.1.92#./.yarn/patches/react-split-pane-npm-0.1.92-93dbf51dff.patch",
     "tsx": "^3.12.7",
-    "typescript": "patch:typescript@npm%3A5.2.2#./.yarn/patches/typescript-npm-5.2.2-01717e9f84.patch"
+    "typescript": "patch:typescript@npm%3A5.3.0-beta#./.yarn/patches/typescript-npm-5.3.0-beta-33b1e90865.patch"
```

Comment:
The future is now old man dot jpg

We can switch this to the 5.3 RC now

Reply:
https://github.com/yarnpkg/berry/issues/4441 keeps hitting me _shakes fist_

Reply:
https://github.com/microsoft/TypeScript/issues/56371 never mind 🥲